azurerm/internal/services/servicefabric/resource_arm_service_fabric_cluster.go

```diff
@@ -273,6 +273,123 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				},
 			},
 
+			"upgrade_description": {
```

Comment:
description doesn't make sense, could we chanege thi to
```suggestion
			"upgrade_policy": {
```